shell/packages/sandstorm-accounts-ui/login_buttons_dropdown.html

```diff
@@ -94,43 +94,23 @@
 </template>
 
 <template name="_loginButtonsLoggedOutPasswordService">
-  {{#if inForgotPasswordFlow}}
-    {{> _forgotPasswordForm}}
-  {{else}}
     <div class="login-form login-password-form">
       {{#each fields}}
         {{> _loginButtonsFormField}}
       {{/each}}
 
-      {{> _loginButtonsMessages}}
-
-      <div class="login-button login-button-form-submit" id="login-buttons-password">
-        {{#if inSignupFlow}}
-          Create account
-        {{else}}
-          Sign in
-        {{/if}}
+      {{#if inSignupFlow}}
+      <div class="login-button login-button-form-submit" id="login-buttons-token">
+        Enter token from email
```

Comment:
The button should just be "Log in".

Above the input box, the heading "Token" should change to "Enter authentication code from login email:"